src/tests/test_stream.cpp

```diff
@@ -53,6 +54,8 @@ class Stream_Cipher_Tests : public Text_Based_Test
             if(nonce.size())
                cipher->set_iv(nonce.data(), nonce.size());
 
+            cipher->seek(seek);
+
```

Comment:
The test should only call if seek is set, since for example RC4 and OFB do not have a fast seek operation and will always throw.